packages/beacon-node/src/util/workerEvents.ts

```diff
@@ -34,6 +40,10 @@ export function wireEventsOnWorkerThread<EventData>(
       // This check is not necessary but added for safety in case of improper implemented events
       isWorkerToMain[data.event] === EventDirection.mainToWorker
     ) {
+      metrics?.networkWorkerWireEventsOnWorkerThreadLatencySec.observe(
+        {eventName: data.event as string},
+        Number(process.hrtime.bigint() - data.posted) / nanoToSecondConversion
```

Comment:
Should be changed to `process.hrtime()`
- see https://github.com/ChainSafe/lodestar/pull/5926